packages/selinux/build.yaml

```diff
@@ -0,0 +1,24 @@
+requires:
+- name: "base"
+  category: "distro"
+  version: ">=0"
+
+{{ if .Values.distribution }}
+{{if eq .Values.distribution "opensuse" }}
+prelude:
+- zypper ar https://download.opensuse.org/repositories/security:/SELinux/openSUSE_Leap_15.3/security:SELinux.repo
+- zypper --gpg-auto-import-keys in -y --allow-vendor-change --allow-downgrade container-selinux -libsemanage1
+
+steps:
+- |
+    mkdir /tmp/rpm && \
+    cd /tmp/rpm && \
+    curl -L -O https://github.com/k3s-io/k3s-selinux/releases/download/v0.3.testing.0/k3s-selinux-0.3-0.el7.noarch.rpm && \
```

Comment:
do we have checksums for this packages somehow?

Reply:
Good point! we do actually https://github.com/k3s-io/k3s-selinux/releases/download/v0.3.stable.0/sha256sum-centos7-noarch.txt

Reply:
done! I've also split the packages separately